pkg/audit/stats_reporter.go

```diff
@@ -0,0 +1,124 @@
+package audit
+
+import (
+	"context"
+	"time"
+
+	"github.com/open-policy-agent/gatekeeper/pkg/metrics"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+const (
+	totalViolationsName  = "violations_total"
+	constraintsTotalName = "constraints_total"
+	auditLatency         = "audit_latency"
+	methodType           = "audit"
+)
+
+var (
+	violationsTotalM  = stats.Int64(totalViolationsName, "Total number of violations per constraint", stats.UnitNone)
+	constraintsTotalM = stats.Int64(constraintsTotalName, "Total number of enforced constraints", stats.UnitNone)
```

Comment:
nit => consider using stats.UnitDimensionless as stats.UnitNone is deprecated

Reply:
thanks, updated